Aliases/distmap@3.1.0-alpha

```diff
@@ -0,0 +1 @@
+../Formula/distmap.rb
```

Comment:
You didn't gave me time for it, but I would recommend to do not have an alias for this: the current version is only for the head, so it will be changing all the time.